{{cookiecutter.github_repository}}/Makefile

```diff
@@ -52,8 +52,11 @@ regenerate:  ## Delete and create new database.
 .PHONY: regenerate
```

Comment:
For ongoing updates in a projects, we can have `make update-libs` command to run `poetry update`. This will update poetry lock to latest libs within bounds mentioned in pyproject.toml

Reply:
I've added `update-libs` command.